src/group/libp2p_group_relcast_server.erl

```diff
@@ -357,7 +360,7 @@ handle_info(force_close, State=#state{}) ->
     %% down the group by exiting the supervisor.
     spawn(fun() ->
                   lager:info("removing group for force_close timeout"),
-                  libp2p_swarm:remove_group(State#state.tid, State#state.group_id)
+                  libp2p_group_mgr:remove_group(State#state.tid, State#state.group_id)
```

Comment:
This was a critical change.   This was allowing some bad races where we'd lose state on restarts.

Reply:
worth just backporting this for now?

Reply:
yes, I think that we should get just this in ASAP.